documentation/docs-templates/steps/checkChangeInDevelopment.md

```diff
@@ -0,0 +1,93 @@
+# checkChangeInDevelopment
+
+## Description
+__STEP_DESCRIPTION__
```

Comment:
I suggest to replace complete sections based on the header instead of using a dedicated tag.